gpii/node_modules/lifecycleManager/src/LifecycleManager.js

```diff
@@ -252,6 +252,13 @@ var gpii = fluid.registerNamespace("gpii");
         }
     };
 
+    /** In the case we are servicing a "restore" rootAction, wrap "dangerous" promises which perform system actions so that
```

Comment:
The long lines in this diff make it inconvenient to read using the GitHub "Unified" diff view. Could you wrap the lines please? Do we have any policy in GPII on max line length?

Looks like the GitHub Unified diff window is max 120 chars wide plus scrollbar. But the scrollbar is off screen if the diff is long. The "Split" diff mode wraps (rather than scrollbar) and I will use that for now.